lib/src/Base/Stat/CovarianceModelImplementation.cxx

```diff
@@ -54,9 +54,10 @@ CovarianceModelImplementation::CovarianceModelImplementation(const UnsignedInteg
   , isDiagonal_(true)
   , isStationary_(false)
   , nuggetFactor_(ResourceMap::GetAsScalar("CovarianceModel-DefaultNuggetFactor"))
-  , activeParameter_(inputDimension_ + outputDimension_)
+  , activeParameter_(0)
 {
-  activeParameter_.fill();
+  for (UnsignedInteger i=0; i < inputDimension_ + 1 + outputDimension_; ++i)
+    if(i != inputDimension_) activeParameter_.add(i);
```

Comment:
what about `i` for  `i >= inputDimension_` ? don't you consider `amplitude` to be active?

Reply:
What these lines are supposed to do is: activate the scale and amplitude, but not the nuggetFactor, which is between the scale and the amplitude in the parameter order.

This is done to preserve the current behavior as the default (scale and amplitude are active, but not the nuggetFactor).